test/exec/child.js

```diff
@@ -1,3 +1,5 @@
 // does nothing child process
 console.log("Child process.id: "+process.pid);
 console.log(" - - - - - - - - - - - - - - - - - - - - - - - ");
+// keep process alive for one minute
+setTimeout(function () {}, 60000);
```

Comment:
As far as the timing of windows tests, I guess the best thing is to make the first test poll until all the child processes are gone, with a test timeout of 5 seconds or so (sometimes it takes more than 1 second for the child processes to go away).